stable/sentry/templates/secrets.yaml

```diff
@@ -22,9 +22,9 @@ data:
   {{ else }}
   user-password: {{ randAlphaNum 16 | b64enc | quote }}
   {{ end }}
-  {{ if and (.Values.postgresql.existingSecret) (or (not .Values.postgresql.enabled) (.Values.postgresql.password)) }}
```

Comment:
The correct variable is `.Values.postgresql.postgresqlPassword`.

Reply:
And I think there should be `and` here, not `or`, like in the case of redis below.